### PR TITLE
feat: Canto EVM

### PR DESCRIPTION
**Pull request by Amurmurmur**
## Description

Add support for Canto EVM-compatible chain

## Checklist

- [ ] Create pull request as draft initially, unless its complete.
- [ ] Add tests to cover changes as needed.
- [ ] Update documentation as needed.
- [ ] If there is a related Issue, mention it in the description.

If you're adding a new blockchain

- [ ] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain.

### CODE REVIEW

File: swift/Podfile.lock [hunk 1]
```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: aac2324ba35cdd5631cb37618cd483887bab9cfd
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.13.0
```
Comment (satoshiotomakan) [inline swift/Podfile.lock:16]: Could you please revert this file?

**Review by satoshiotomakan — CHANGES_REQUESTED** [review]
Thank you for opening the PR! I have only one note

**Comment by satoshiotomakan** [comment]
Hi @Amurmurmur, could you please fix the merge conflicts?

**Comment by Milerius** [comment]
Not merging because of: https://twitter.com/CantoPublic/status/1703809950290718871

**Pull request closed without merging**